testbed/testbed/senders.go

```diff
@@ -129,15 +144,54 @@ func (je *JaegerDataSender) GenConfigYAMLStr() string {
       thrift-binary:
         endpoint: "localhost:8374"
       thrift-http:
-        endpoint: "localhost:%d"`, je.port)
+        endpoint: "localhost:%d"`, je.Port)
 }
 
-func (je *JaegerDataSender) GetCollectorPort() int {
-	return je.port
+func (je *JaegerThriftDataSender) ProtocolName() string {
+	return "jaeger"
 }
 
-func (je *JaegerDataSender) ProtocolName() string {
-	return "jaeger"
+// OCTraceDataSender implements TraceDataSender for OpenCensus trace protocol.
+type OCTraceDataSender struct {
+	DataSenderOverTraceExporter
+}
+
+// Ensure OCTraceDataSender implements TraceDataSender.
+var _ TraceDataSender = (*OCTraceDataSender)(nil)
+
+// NewOCTraceDataSender creates a new OCTraceDataSender that will send
+// to the specified port after Start is called.
+func NewOCTraceDataSender(port int) *OCTraceDataSender {
+	return &OCTraceDataSender{DataSenderOverTraceExporter{Port: port}}
+}
+
+func (ote *OCTraceDataSender) Start() error {
+	cfg := &opencensusexporter.Config{
+		GRPCSettings: configgrpc.GRPCSettings{
+			Endpoint: fmt.Sprintf("localhost:%d", ote.Port),
+		},
+	}
+
+	factory := opencensusexporter.Factory{}
+	exporter, err := factory.CreateTraceExporter(zap.L(), cfg)
+
```

Comment:
nit: extra whie-line: typically we have the err check right after the statement setting `err`